script/updateManifest.cjs

```diff
@@ -37,8 +37,8 @@ fs.readFile(manifestPath, 'utf-8', (err, data) => {
     .replace(
       /<Menu>(.*?)<\/Menu>/,
       isDev
-        ? '<Menu>Plainly plugin - Dev mode</Menu>'
-        : '<Menu>Plainly plugin</Menu>',
+        ? '<Menu>Plainly Videos - Dev mode</Menu>'
+        : '<Menu>Plainly Videos</Menu>',
```

Comment:
@ivansenic from your PR, needs to be changed here also, so `yarn build/dev` produce the same name